### PR TITLE
Added CRUD API endpoints and DB queries

### PR DESCRIPTION
**Pull request by creationator**
This is just an update to my README file and added my API endpoints along with the necessary DB queries. Consider these queries a WIP.

### CODE REVIEW

File: server/server.js [hunk 1]
```diff
@@ -15,6 +15,8 @@ app.use(morgan('dev'));
 
 app.use('/products/:id', express.static(__dirname + '/../dist'));
 
+app.post('/')
+
 app.get('/product/:id', (req, res) => {
```
Comment (Gosuguy) [inline server/server.js]: Can all of this into its own 'controller' file

File: server/server.js [hunk 2]
```diff
@@ -23,11 +25,40 @@ app.get('/product/:id', (req, res) => {
 
 app.get('/data/grab', (req, res) => {
   dbConnection.connection.query('SELECT * FROM photos', (err, data) => {
-    if(err){return console.log(err, 'err')}
+    if(err){return cconsole.error(err, 'err')}
```
Comment (Gosuguy) [inline server/server.js]: cconsole typo

File: server/server.js [hunk 3]
```diff
@@ -23,11 +25,40 @@ app.get('/product/:id', (req, res) => {
...
     res.send(data);
   })
 })
 
+app.post('/product/:id', (req, res) => {
+  dbConnection.connection.query('INSERT INTO xx_BLOB(IMAGE) VALUES(LOAD_FILE('someImage.jpg'))', (req, res) => {
```
Comment (auramix) [inline server/server.js]: Your db.js file seems to have two query functions defined but you don't seem to be using them in your CRUD API? You might consider adding the queries you use here in your server file as functions to be exported from db.js

Comment (auramix) [inline server/server.js]: Are you attempting to save image files, rather than links to files in your MySQL db?

**Pull request merged**